crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -138,7 +142,7 @@ en:
       confirm_delete_proposal: 'Are you sure you wish to delete this proposal?'
       available_clusters: 'Available Clusters'
       available_clusters_hint: 'Here you can see all available clusters.'
-      available_remotes: 'Available Remotes'
+      available_remotes: 'Available Clusters with Remote Nodes'
```

Comment:
Actually, if you look at the UI, it is relatively consistent. It's just that the strings don't appear in the same kind of labels. But if you find any such issue, yes, we should fix that! (it's a thing I find very annoying too)